openmc/mgxs/__init__.py

```diff
@@ -6,26 +6,32 @@
 from openmc.mgxs.mdgxs import *
 
 GROUP_STRUCTURES = {}
-"""Dictionary of commonly used energy group structures, including "CASMO-X" (where X
-is 2, 4, 8, 16, 25, 40 or 70) from the CASMO_ lattice physics code and other commonly
-used activation_ energy group structures "VITAMIN-J-175", "TRIPOLI-315", "CCFE-709_" 
-and "UKAEA-1102_"
+"""Dictionary of commonly used energy group structures:
+- "CASMO-X" (where X is 2, 4, 8, 16, 25, 40 or 70) from the CASMO_ lattice
+physics code
```

Comment:
sphinx is picky about the format here. You need to keep this line indented so that it knows it's part of the above bullet point:
```rst
- "CASMO-X" (where X is ...
  physics code
- "XMAS-172_" designed ...
```

Reply:
I checked by rebuilding the documentation and I think this time it works